test-engineering/load-tests/setup_k8.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+set -eu
+
+#Declare variables
+GCLOUD=$(which gcloud)
+SED=$(which sed)
+KUBECTL=$(which kubectl)
+GOOGLE_CLOUD_PROJECT=$(gcloud config get-value project)
+CLUSTER=merino-locust-load-test
+TARGET='https://stage.merino.nonprod.cloudops.mozgcp.net'
+SCOPE='https://www.googleapis.com/auth/cloud-platform'
+REGION=us-central1
```

Comment:
```suggestion
REGION='us-central1'
```

Reply:
Quotes are not required. I have only used quotes to declare the urls.